Code/CoreData/RKManagedObjectStore.m

```diff
@@ -191,12 +207,6 @@ - (instancetype)initWithPersistentStoreCoordinator:(NSPersistentStoreCoordinator
     return self;
 }
 
-- (instancetype)init
```

Comment:
We should be ok overriding init and calling initWithManagedObjectModel here. Reverting this change.